src/components/documentComponents/documentLinkSearch/ContentInnerModal.js

```diff
@@ -0,0 +1,51 @@
+import CommandList from "./CommandList.js";
+import DocumentLinkList from "./DocumentLinkList.js";
+
+export default function ContentInnerModal({ $target, selectionStart, option }) {
+  const $contentInnerModal = document.createElement("div");
+  $contentInnerModal.className = "content-inner-modal";
+  $contentInnerModal.contentEditable = false;
+  console.log(selectionStart);
+  if (selectionStart < 28) {
+    $contentInnerModal.style.left = `${selectionStart === 1 || 6 ? 0 : selectionStart + 60}px`;
+  } else {
+    $contentInnerModal.style.right = `${selectionStart + 22}px`;
+  }
+
+  $target.appendChild($contentInnerModal);
+
+  this.close = () => {
+    $contentInnerModal.remove();
+  };
+
+  if (option === "command") {
+    return new CommandList({
+      $parent: $target,
+      $target: $contentInnerModal,
+      onClose: () => {
+        const $textNode = $target.childNodes[0];
+        if ($textNode && $textNode.nodeType === Node.TEXT_NODE) {
+          const $text = $textNode.nodeValue;
+          const extractedText = $text.slice(-6);
+          const newText = $text.slice(0, -extractedText.length);
+          $textNode.nodeValue = newText;
+        }
+
+        this.close();
+      },
+    });
```

Comment:
이렇게 함수를 생성하고 할당하는 것도 좋을 것 같습니다.

```suggestion
  const onCloseForCommand = () => {
    const $textNode = $target.childNodes[0];
    if ($textNode && $textNode.nodeType === Node.TEXT_NODE) {
      const $text = $textNode.nodeValue;
      const extractedText = $text.slice(-6);
      const newText = $text.slice(0, -extractedText.length);
      $textNode.nodeValue = newText;
    }

    this.close();
  };
// 생략
onClose: onCloseForCommand,
```